sway-lsp/src/capabilities/code_actions/struct_decl/mod.rs

```diff
@@ -0,0 +1,25 @@
+pub(crate) mod struct_impl;
+pub(crate) mod struct_new;
+
+use sway_core::decl_engine::DeclId;
+use sway_types::Spanned;
+use tower_lsp::lsp_types::CodeActionOrCommand;
+
+use self::{struct_impl::StructImplCodeAction, struct_new::StructNewCodeAction};
+
+use super::{CodeActionContext, CodeActionTrait};
```

Comment:
i'm not a fan of using super but it's down to personal preference, up to you if you want to leave it like this

Reply:
What's the alternative?